c2corg_api/views/validation.py

```diff
@@ -359,12 +371,13 @@ def _is_parent_of_association(main_document_type, other_document_type):
 # associations that can be updated/created when updating/creating a document
 # e.g. when creating a route, route and waypoint associations can be created
 updatable_associations = {
-    ROUTE_TYPE: {'articles', 'routes', 'waypoints'},
+    ROUTE_TYPE: {'articles', 'routes', 'waypoints', 'reports'},
```

Comment:
If the reports are not included for routes (and the other types, see my comment above), they should also not be updatable. To confirm with @asaunier.